src/Components/Components/ref/Microsoft.AspNetCore.Components.netcoreapp.cs

```diff
@@ -107,6 +107,11 @@ public abstract partial class ComponentBase : Microsoft.AspNetCore.Components.IC
         protected virtual bool ShouldRender() { throw null; }
         protected void StateHasChanged() { }
     }
+    public partial class DefaultComponentActivator : Microsoft.AspNetCore.Components.IComponentActivator
```

Comment:
Does this need to be public? Channeling my inner rynowak, should we just have an Abstract Base Class here?